### PR TITLE
De bugging lamino session

### PR DESCRIPTION
**Pull request by edobN**
Corrected line 55, which gave an error:
BLISS lamino-session contains frontend shutter as "frontend" not "frontendDevice"

### CODE REVIEW

File: esrfconcert/experiments/laminography.py [hunk 1]
```diff
@@ -14,170 +14,21 @@
...
 
 LOG = logging.getLogger(__name__)
 
 
-class Laminography(Experiment):
-    num_flats = Parameter()
-    num_darks = Parameter()
-    num_projections = Parameter()
-    radio_position = Quantity(q.deg)
-    flat_position = Quantity(q.deg)
```
Comment (tfarago) [inline esrfconcert/experiments/laminography.py:27]: @tfarago we need to fix the unit.

Comment (tfarago) [inline esrfconcert/experiments/laminography.py:27]: @tfarago we need to fix the unit.

**Comment by MarcusZuber** [comment]
Btw, if not to much unexpected stuff appears, I will put the general lamino stuff in concert tomorrow. (What I sent you in January)

File: esrfconcert/experiments/laminography.py [hunk 2]
```diff
@@ -64,6 +64,8 @@ def __init__(self, walker, flat_motor, scanning_motor, shutter, radio_position,
                                                      camera=camera, num_flats=num_flats, num_darks=num_darks,
                                                      num_projections=num_projections, angular_range=angular_range,
                                                      start_angle=start_angle, separate_scans=separate_scans)
+        self['radio_position']._parameter.unit = q.deg
+        self['flat_position']._parameter.unit = q.deg
```
Comment (tfarago) [inline esrfconcert/experiments/laminography.py:78]: @MarcusZuber I had a look in ankaconcert and you "overwrote" these in the mixin class. We should think of a clean way how to deal with these.

Comment (MarcusZuber) [reply esrfconcert/experiments/laminography.py:78]: What's the problem with overwriting them?

Mixing classes are always quite ugly if you do more than just defining interfaces...

**Comment by tfarago** [comment]
@edobN please check this once again. I squashed the de-bugging commits (i.e. merged them to just one) and hopefully didn't mess up anything... Then we can merge.

**Pull request merged**